test/integrationtests/voight_kampff/tools.py

```diff
@@ -15,15 +15,161 @@
 """Common tools to use when creating step files for behave tests."""
 
 from threading import Event
+from typing import Any, Callable, List, Tuple
 import time
 
 from mycroft.audio.utils import wait_while_speaking
 from mycroft.messagebus import Message
 
-
 DEFAULT_TIMEOUT = 10
 
 
+class VoightKampffEventMatcher:
+    """Matches a specified message type to messages emitted on the bus.
+
+    Attributes:
+        message_type: identifier of the message to search for on the bus
+        context: the Behave context from the test utilizing this class
+        match_event: mechanism for knowing when a match is found
+        error_message: message that can be used by the test to communicate
+            the reason for a failed match to the tester.
+    """
+    def __init__(self, message_type: str, context: Any):
+        self.message_type = message_type
+        self.context = context
+        self.match_event = Event()
+        self.match_found = False
```

Comment:
`match_found` is a good candidate to be a property since it reflects the state of the `match_event` Event.

Reply:
Good call. Will do.